packages/theme/pages/MyAccount/AddressesDetails.vue

```diff
@@ -120,6 +120,16 @@ export default defineComponent({
     const getTranslatedUrlAddress = (title) => app.i18n.t(`${title}`).toLowerCase().replace(' ', '-');
     const isNewAddress = computed(() => route.value.query.id === 'new');
     const editingAddress = computed(() => !!route.value.query.id);
+    const { fetch } = useFetch(async () => {
+      const addressesData = await load();
+      userAddresses.value = userAddressesGetters.getAddresses(addressesData);
+
+      const activeAddressData = userAddresses.value
+        .filter((address) => String(address?.id) === route.value.query.id)
+        .pop();
```

Comment:
```suggestion
      const [ ..._front, activeAddressData] = userAddresses.value
        .filter((address) => String(address?.id) === route.value.query.id);
```

```suggestion
      const [ ...front, activeAddressData] = userAddresses.value
        .filter((address) => String(address?.id) === route.value.query.id).at(-1);
```

.pop() seems ununtuitive to me here, proposed some more explicit ways

Reply:
thanks

Reply:
but after change you requested code stopped work as excepted